dev/README_RELEASE_AIRFLOW.md

```diff
@@ -806,12 +806,16 @@ Documentation for providers can be found in the ``/docs/apache-airflow`` directo
     ```shell script
     ./docs/publish_docs.py --package-filter apache-airflow --package-filter docker-stack
     cd "${AIRFLOW_SITE_DIRECTORY}"
+    cd post-docs
+    python add-back-references.py airflow
+    cd ..
```

Comment:
See the readme: https://github.com/apache/airflow-site/tree/main/post-docs - implemented by @amoghrajesh

Reply:
Basically - whenever we add new files or redirects , they work fine for redirecting when you have old links, byt when you look at the docs in the new version and change the version (from left-top menu) you get "not-found". This scripts generates automatically redirect files in the old versions of the docuementation in our repo to fix it. This is what I've used (after few fixes) when we generated changelog/security pages. 

https://github.com/apache/airflow-site/pull/831

Here my fix to generate those: https://github.com/apache/airflow-site/pull/830

Shortly @amoghrajesh will move this script to a new `breeze release-management` command and remove it from "airlfow-site" (and update the RELEASE* docs, so that we can do everything from a single `breeze` environment.

Reply:
Thanks to that one, when you look at https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/changelog.html and change the version of amazon provider to some past version, you will be redirected to "index" rather than getting a "Not found" error.

Reply:
I understand the concept of what add references does.
My question is about cd post-doc.
I dont understand what is the command to run.
What is the post doc folder?

Reply:
This folder in `airflow-site` https://github.com/apache/airflow-site/tree/main/post-docs

Reply:
I may miss something... Did the script create post-doc folder?

Reply:
Yes, as part of the back reference PR, I had the script present in the post_doc folder by design

Reply:
`post-docs` is where the scripts are :)